scripts/contributor-workflow-setup.sh

```diff
@@ -94,7 +94,7 @@ git commit -m "Update README" --no-gpg-sign
 # Push commit to monorepo
 rad push
 # Create patch
-rad patch --sync
+rad patch --sync --message 'title' --message "description"
```

Comment:
```suggestion
rad patch --sync --message 'Update README' --message "No description..."
```

Reply:
Will be useful to have a `--no-edit` flag next, so that it just re-uses the commit message without asking for input.